citizenvoice/survey_design/models/answer.py

```diff
@@ -0,0 +1,71 @@
+"""
+    These type-specific answer models use a text field to allow for flexible
```

Comment:
Whenever we reuse code from other sources, as I think is the case here. Please mention that at the top of the file and credit the original authors. You can add something like, "This source code is based on <App/project>  by <authors>. <Link to the original repo/source>